### PR TITLE
Add apt_package_security_updates fact

### PR DESCRIPTION
**Pull request by jle64**
Add a apt_package_security_updates fact, similar to apt_package_updates but which only lists security updates.

### CODE REVIEW

File: lib/facter/apt_updates.rb [hunk 1]
```diff
@@ -42,6 +42,17 @@
   end
 end
 
+Facter.add("apt_package_security_updates") do
```
Comment (tphoney) [inline lib/facter/apt_updates.rb]: How is this different to lines 63-67, apologies if i am missing something obvious. No caffeine yet.

**Comment by jle64** [comment]
Lines 63-67 (fact apt_security_updates) are about the number of security updates (it returns apt_package_updates[1].length), this adds the list of security updates.

**Comment by tphoney** [comment]
@jle64 Yup i agree, totally makes sense. 👍  Apologies for what i am about to ask

Could you please have a look at replicating the facter tests https://github.com/puppetlabs/puppetlabs-apt/blob/bd3b3fbe990bfa824ebed3504c0c82175d7c7ec9/spec/unit/facter/apt_package_updates_spec.rb for this new fact.

With this new fact, could you add a line or two to the readme explaining this cool new thing

**Comment by tphoney** [comment]
Thanks for the PR #703  is merged. I just added a test.

**Pull request closed without merging**